model/const.go

```diff
@@ -160,6 +165,10 @@ const (
 	// SignTypeV2 is an authentication algorithm, which is used by metamask
 	SignTypeV2 = "authTypeV2"
 
+	SignTypeOffChain   = "OffChainAuth" // sign type - off-chain-auth
```

Comment:
When to remove SignTypeV2?

Reply:
After Dcellar uses off-chain-auth, I will remove SignTypeV2.  So far, let's keep it and leave some time for Dcellar